cypress/tests/core/basic/groups-controlpanel.js

```diff
@@ -1,11 +1,11 @@
-const interceptGroups = () => {
-  cy.intercept('/**/usergroup').as('manyGroups');
-  cy.visit('/controlpanel/groups');
-  cy.waitForResourceToLoad('@navigation');
-  cy.waitForResourceToLoad('@breadcrumbs');
-  cy.waitForResourceToLoad('@actions');
-  cy.waitForResourceToLoad('@types');
-};
+// const interceptGroups = () => {
```

Comment:
I will revisit the entire cypress suite for this during the conference sprint. Just too busy here with holidays these days. Sorry for trouble!

Reply:
@nileshgulia1 no problem at all! thanks for caring, enjoy the holidays and see you soon!